### PR TITLE
[fix](recycler) fix idx file leak in recycler

### PR DESCRIPTION
**Pull request by airborne12**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #38306

Problem Summary:
Non inverted index storage format for old version will cause leaking recycling idx files.
### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [x] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by airborne12** [comment]
run buildall

File: cloud/src/recycler/recycler.cpp [hunk 1]
```diff
@@ -1430,7 +1436,7 @@ int InstanceRecycler::delete_rowset_data(const std::vector<doris::RowsetMetaClou
         // Process inverted indexes
         std::vector<std::pair<int64_t, std::string>> index_ids;
         // default format as v2.
-        InvertedIndexStorageFormatPB index_format = InvertedIndexStorageFormatPB::V2;
+        InvertedIndexStorageFormatPB index_format = InvertedIndexStorageFormatPB::V1;
```
Comment (gavinchou) [inline cloud/src/recycler/recycler.cpp:1439]: wrong comment

**Comment by airborne12** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by csun5285 — APPROVED** [review]
LGTM

**Pull request merged**